Sources/Noora/Components/SingleChoicePrompt.swift

```diff
@@ -20,6 +20,7 @@ struct SingleChoicePrompt {
     let terminal: Terminaling
     let collapseOnSelection: Bool
     let filterMode: SingleChoicePromptFilterMode
+    let autoSelectSingle: Bool
```

Comment:
```suggestion
    let autoselectSingleChoice: Bool
```

I understand `autoselect`, so it imho reads better with that capitalization, but that's quite subject and minor, so feel free to keep as-is. I'd add the `Choice` suffix to be a bit more explicit, but again, subjective. Naming is hard 😅 

I'd add the